envbuilder.go

```diff
@@ -518,11 +522,42 @@ func Run(ctx context.Context, options Options) error {
 				RegistryMirrors: registryMirror,
 			},
 			SrcContext: buildParams.BuildContext,
-		})
+
+			// For cached image utilization, produce reproducible builds.
+			Reproducible: options.PushImage,
```

Comment:
Where is this getting used?

Reply:
See https://github.com/coder/kaniko/pull/12